cmd/query/app/querysvc/query_service_test.go

```diff
@@ -115,6 +116,68 @@ func TestGetTraceSuccess(t *testing.T) {
 	assert.Equal(t, res, mockTrace)
 }
 
+func TestGetTraceWithRawTraces(t *testing.T) {
```

Comment:
@yurishkuro do we want to add this test to all the handlers as well? or is it sufficient to just have it in the query service?

Reply:
follow encapsulation - the change in behavior is only in QS and should be tested in QS. The handlers only pass the flag, so their tests only need to test passing the flag, not whether QS respects that flag.

Reply:
@yurishkuro I'm not sure if there's a clean way to do that at the moment because the query service isn't mocked out in the tests. The tests are mocking out the storage interface.

Reply:
that's fine, my point is that the tests you added here for sorted tags do not need to be repeated in the handlers.